WalletConnectSharp.Core/Controllers/Relayer.cs

```diff
@@ -199,33 +199,46 @@ protected virtual Task<IJsonRpcConnection> BuildConnection(string url)
 
         protected virtual void RegisterProviderEventListeners()
         {
-            Provider.RawMessageReceived += (sender, s) =>
-            {
-                OnProviderPayload(s);
-            };
+            Provider.RawMessageReceived += ProviderOnRawMessageReceived;
+            Provider.Connected += ProviderOnConnected;
+            Provider.Disconnected += ProviderOnDisconnected;
+            Provider.ErrorReceived += ProviderOnErrorReceived;
```

Comment:
Let's try to be consistent with naming when possible. 

`ProviderOnRawMessageReceived` →  `OnProviderRawMessageReceived`